default/stringtables/en.txt

```diff
@@ -5032,7 +5032,14 @@ SITREP_SNOOZE_CLEAR_INDEFINITE
 '''  Show all SitReps indefinitely dismissed'''
 
 SITREP_SNOOZE_CLEAR_ALL
-'''  Show all hidden or dismissed SitReps'''
+'''  Show all dismissed SitReps'''
+
+# %1% localized name of SitRep template
+SITREP_HIDE_TEMPLATE
+'''  Hide all SitReps with template: %1%'''
```

Comment:
Agreed: we use the term 'label' in the scripts and having been thinking about it for a few days I haven't come up with a more appropriate term.